code/modules/shuttle/shuttle.dm

```diff
@@ -278,6 +278,8 @@
 /obj/docking_port/mobile/proc/canDock(obj/docking_port/stationary/S)
 	if(!istype(S))
 		return SHUTTLE_NOT_A_DOCKING_PORT
+	if(S.ignore_bounds == 1) // Does the port ignore size bounds
+		return SHUTTLE_CAN_DOCK	
```

Comment:
So what exactly will it mean if an oversized shuttle tries to dock on a port where it won't fit?

And what will happen if there's another shuttle docked at the port?

And what will happen if the shuttle attempts to transit to the port it is already docked at?

Reply:
I think it might help to write out the way `width`, `dwidth`, `height`, and `dheight` interact in a notebook, before you tamper with this proc too much. As-is, it seems like you're short-circuiting a few important checks

Reply:
Ill move the checks around a bit, but having a look, if the shuttle is not the **exact** size, it doesnt work properly